### DISCUSSION TITLE
No tabular data for GTFS

### DISCUSSION THREAD

**Discussion by mpaine-act** [frictionlessdata/frictionlessdata.io#807] [Q&A]
I'm trying to understand why fictionless returns an error trying to extract from GTFS.zip:

`
PS C:\tmp> frictionless validate *
───────────────── Dataset ─────────────────
┏━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ name            ┃ type ┃ path                ┃ status ┃
┡━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ agency          │ text │ agency.txt          │ VALID  │
│ calendar        │ text │ calendar.txt        │ VALID  │
│ calendar_dates  │ text │ calendar_dates.txt  │ VALID  │
│ fare_attributes │ text │ fare_attributes.txt │ VALID  │
│ fare_rules      │ text │ fare_rules.txt      │ VALID  │
│ feed_info       │ text │ feed_info.txt       │ VALID  │
│ routes          │ text │ routes.txt          │ VALID  │
│ shapes          │ text │ shapes.txt          │ VALID  │
│ stops           │ text │ stops.txt           │ VALID  │
│ stop_times      │ text │ stop_times.txt      │ VALID  │
│ trips           │ text │ trips.txt           │ VALID  │
└─────────────────┴──────┴─────────────────────┴────────┘
PS C:\tmp> frictionless describe *
───────────────── Dataset ─────────────────
┏━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━━━━━━━━━━━━┓
┃ name            ┃ type ┃ path                ┃
┡━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━━━━━━━━━━━━┩
│ agency          │ text │ agency.txt          │
│ calendar        │ text │ calendar.txt        │
│ calendar_dates  │ text │ calendar_dates.txt  │
│ fare_attributes │ text │ fare_attributes.txt │
│ fare_rules      │ text │ fare_rules.txt      │
│ feed_info       │ text │ feed_info.txt       │
│ routes          │ text │ routes.txt          │
│ shapes          │ text │ shapes.txt          │
│ stops           │ text │ stops.txt           │
│ stop_times      │ text │ stop_times.txt      │
│ trips           │ text │ trips.txt           │
└─────────────────┴──────┴─────────────────────┘
───────────────── Tables ──────────────────
PS C:\tmp> frictionless extract *
╭─ Error ─────────────────────────────────╮
│ No tabular data have been found in the source                                                                                                                                                                 │
╰──────────────────────────────────────╯

`

You can see the data I used @ [link](https://api.actransit.org/transit/gtfs/download?token=2512B81107A09D2DC44895CDDC650D47)

**Comment by roll** [comment]
Thanks @mpaine-act! I've created a bug report -- we will investigate

**Comment by shashigharti** [comment]
@mpaine-act The reason for the error is the file format mentioned [here](https://github.com/frictionlessdata/framework/issues/1492). I have created a new issue to support GTFS format [here](https://github.com/frictionlessdata/framework/issues/1497).
